src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -574,7 +574,7 @@ jsBackend.pages.extras = {
             $('.templatePositionCurrentType[data-block-id=' + index + '] .editUserTemplate').click()
           }
         }).modal('hide')
-        
+
```

Comment:
Seems like there was nothing changed in this commit? Squash it with the previous one.